### PR TITLE
Chore: Quick Troubleshooting guide

### PR DESCRIPTION
**Pull request by acekyd**
## What type of PR is this? (check all applicable)

- [x] 📝 Documentation Update

## Description

This PR updates the troubleshooting guide. 

## Related Tickets & Documents

Resolves #

## Added code snippets?
- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### CODE REVIEW

**Review by blackgirlbytes — APPROVED** [review]
love it!

File: site/docs/web5/troubleshooting/vue.mdx [hunk 1]
```diff
@@ -113,7 +115,7 @@ export default defineConfig({
 
 If you run into any errors, check out our [troubleshooting guide](./common-errors) for a possible solution.
 
-## Uncaught ReferenceError: require is not defined
+## Uncaught ReferenceError: xxxx is not defined
```
Comment (blackgirlbytes) [inline site/docs/web5/troubleshooting/vue.mdx:118]: why change to this? does it just throw general errors that any value is undefined?

Comment (acekyd) [reply site/docs/web5/troubleshooting/vue.mdx:118]: Yes.. I have had 3 different occurrences and the fix in the docs doesn't work for all of them.

Comment (blackgirlbytes) [reply site/docs/web5/troubleshooting/vue.mdx:118]: gotcha!

**Review by angiejones — APPROVED** [review]

**Pull request merged**